ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -125,11 +125,16 @@
             ConfigDef.Importance.MEDIUM,
             "The default number of replicas for the topics created by KSQL."
             )
-    .define(SINK_WINDOW_CHANGE_LOG_ADDITIONAL_RETENTION_PROPERTY,
+    .define(SINK_WINDOW_CHANGE_LOG_ADDITIONAL_RETENTION_MS_PROPERTY,
             ConfigDef.Type.LONG,
             defaultSinkWindowChangeLogAdditionalRetention,
             ConfigDef.Importance.MEDIUM,
-            "The default window change log additional retention time."
+            "The default window change log additional retention time. This is a streams "
+            + "config value which will be added to a windows maintainMs to ensure data is not "
+            + "deleted from "
+            + "the "
```

Comment:
Why these new lines?

Reply:
Fixed it, seems that my intelliJ formatted it that way automatically.